storage/ndb/rest-server/rest-api-server/internal/config/structs.go

```diff
@@ -95,6 +126,20 @@ func (m Mgmd) Validate() error {
 
 type RonDB struct {
 	Mgmds []Mgmd
+
+	// Connection pool size. Default 1
+	// Note current implementation only supports 1 connection
+	// TODO JIRA RonDB-245
+	ConnectionPoolSize uint32
+
+	// This is the list of node ids to force the connections to be assigned to specific node ids.
+	// If this property is specified and connection pool size is not the default,
+	// the number of node ids must match the connection pool size
+	NodeIDs []uint32
```

Comment:
You mean API node Ids? Could you specify that? (Could also be mgmd ids afaik).

Reply:
Can we just add a comment to the C code, but rename the config, since that is our API? Also, you can use .jsonc files to add comments. Check if there are available Go libraries to parse that.

Reply:
Well last time I checked you used `storage/ndb/rest-server/rest-api-server/README.md` to describe the configurations.

Reply:
fixed